Sources/NautilusTelemetry/Tracing/Span+URLSession.swift

```diff
@@ -164,6 +182,19 @@ public extension Span {
 		}
 	}
 
+	/// Returns the name / value pair for the traceparent header
+	/// - Parameter sampled: Whether we are sampling
+	/// - Returns: a traceparent header
+	internal func traceParentValue(sampled: Bool) -> (String, String) {
```

Comment:
Maybe `traceParentHeaderValue` to communicate it returns a (header name, value) pair?